packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/TaskService.java

```diff
@@ -87,16 +91,19 @@ private static class TaskRequestWithId extends TaskRequest {
 			userId = req.getUserId();
 			projectId = req.getProjectId();
 			timeoutMinutes = req.getTimeoutMinutes();
+			notificationEventType = req.getNotificationEventType();
 			additionalProperties = req.getAdditionalProperties();
 		}
 
 		public TaskResponse createResponse(final TaskStatus status) {
 			return new TaskResponse()
 					.setId(id)
 					.setStatus(status)
+					.setType(type)
 					.setUserId(userId)
 					.setProjectId(projectId)
 					.setScript(getScript())
+					.setNotificationEventType(notificationEventType)
```

Comment:
I think a better way to deal with this is to set the `type` on the group to the `getScript` of the TaskRequest / TaskResponse. We don't need a separate ClientEventType on the taskrunner stuff.

Reply:
Done